aether-ui/aether/ui/assets/apps/pipeline/components/PipelineRename.jsx

```diff
@@ -54,10 +54,10 @@ class PipelineRename extends Component {
         onEscape={this.props.onCancel}
         buttons={buttons}
         header={
-          <FormattedHTMLMessage
-            defaultMessage='Rename pipeline <b>{name}</b>'
+          <FormattedMessage
+            defaultMessage='Rename pipeline {name}'
             id='rename.modal.header'
-            values={{ name: this.props.name }}
+            values={{ name: <b>{this.props.name}</b> }}
```

Comment:
Please, leave the `<b>` tag within the default message. It's more clear. :nerd_face:

Reply:
But that will mean the `<b>` tag will have to be resolved in the `values` again 🤷‍♂

Reply:
You are right! :thinking: 

OK, let's leave it as it is now if you prefer so.

Reply:
I think with `FormattedHTMLMessage` the tags should be in the default message, but for `FormattedMessage` if the styling can be achieved in just the values that's the way to go to avoid resolving the html tags defined in the default message in the values AGAIN ==> extra complexity